.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Release
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-java@v1
+      with:
+        java-version: '12.x'
+    - uses: subosito/flutter-action@v1
+      with:
+        flutter-version: '1.20.4'
+    - run: flutter pub get
+    - run: flutter test
+    - run: flutter build apk --split-per-abi
+    - run: flutter build ios --release --no-codesign
+    - name: Create Release APK
+      uses: ncipollo/release-action@v1
+      with:
+        artifacts: "build/app/outputs/apk/debug/*.apk"
+        token: ${{ secrets.TOKEN }}
```

Comment:
Need to set this github token but *in theory* this should build/bundle APKs on new tags/releases. We'll need to play with this but will be nice to have those automatically built, especially for iOS if we can.